### DISCUSSION TITLE
lastUpdated requires git?

### DISCUSSION THREAD

**Discussion by jiaoHsiung** [vuepress/core#726] [Q&A] [answered]
lastUpdatedText is set, but not set git, the field is not working.
Do both of them have to be set to work?
![image](https://user-images.githubusercontent.com/24486922/155959315-edf34a68-c2fd-4a19-ae84-34c42ca1ebc2.png)

**Comment by Mister-Hope** [comment, marked as answer]
~~Reproduction repo? This option is working as expected in my env.~~

LastUpdated feature is built on git becuz the update timestamp is generated from git.

BTW, use a proper title next time, I though you are reporting this option not working at the first glance.

**Reply by jiaoHsiung** [reply]
好的，但是我仍然有疑问，git 不是默认开启的吗，为什么我不配置，lastUpdated仍然没有显示呢？

**Reply by Mister-Hope** [reply]
默认主题是默认启用的。

这需要注意的是：你需要生成信息的文件必须已经被提交到仓库中，而不是 untracked 或者只提交到了暂存区。

如果你觉得仍有问题，提供一个minimal reproduction repo link。

**Reply by jiaoHsiung** [reply]
多谢 没问题了

**Reply by jiaoHsiung** [reply]
哈喽 我发现一个问题 当结构变成了下图 lastUpdated 又不起作用了 这是怎么回事呢？
![image](https://user-images.githubusercontent.com/24486922/156548963-0e7031b3-56ed-4c53-afeb-f52b001df035.png)

**Reply by Mister-Hope** [reply]
原回答已更改。
